core/templates/components/summary-tile/exploration-summary-tile.component.html

```diff
@@ -170,10 +176,6 @@ <h3 class="mobile-exploration-title protractor-test-exp-summary-tile-title">
     margin-left: 6px;
   }
 
-  .protractor-test-exp-summary-tile-title {
-    font-size: 1.7rem;
-  }
```

Comment:
Thanks for removing this, it seems odd that we added style to a protractor specific class -- these classes are just meant for e2e tests. It may be a good idea to have a lint rule to prevent adding of styles to protractor classes so things like this don't slip through, what do you think?

Reply:
That sounds like a good idea, although I'm not sure how we should proceed (perhaps I should open an issue to track it)?

Reply:
Yes, please create an issue.

Reply:
[Done.](https://github.com/oppia/oppia/issues/15636)